import-secrets/handler.go

```diff
@@ -67,12 +67,12 @@ func Handle(req []byte) string {
 		return fmt.Sprintf("invalid owner name %s", event.owner)
 	}
 
-	if strings.HasPrefix(userSecret.Metadata.Name, event.owner) == false {
+	name := strings.ToLower(fmt.Sprintf("%s", userSecret.Metadata.Name))
```

Comment:
Why do we need sprint here?

Reply:
🤷‍♀️ it was like this before hehe

but removed it 💃